airbyte-cdk/bulk/core/extract/src/testFixtures/kotlin/io/airbyte/cdk/discover/ResourceDrivenMetadataQuerierFactory.kt

```diff
@@ -73,6 +73,9 @@ class ResourceDrivenMetadataQuerierFactory(
             }
 
             override fun extraChecks() {}
+            override fun commonCursorOrNull(cursorColumnID: String): FieldOrMetaField? {
```

Comment:
would this throw error if not implemented?

Reply:
Yes. Implementing
I just wanted to consult about the direction